packages/react-components/react-combobox/library/tsconfig.cy.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "isolatedModules": false,
     "types": ["node", "cypress", "cypress-real-events"],
+    "typeRoots": ["../../../../node_modules", "../../../../node_modules/@types"],
```

Comment:
this is indeed unfortunate change. 
Fortunately it has a better solution in TS 5.5 

while we wanna migrate gradually let's track it under an issue that this will be reverted once we land 5.5  by this feature https://devblogs.microsoft.com/typescript/announcing-typescript-5-5/#the-${configdir}-template-variable-for-configuration-files.


please make sure to update cypress generator to reflect these changes https://github.com/microsoft/fluentui/tree/master/tools/workspace-plugin/src/generators/cypress-component-configuration

Reply:
Yeah, I know it's not ideal, but the only way for ts 5.2 to work right now. Will add this to PR description, thanks!